rerun.go

```diff
@@ -183,6 +186,8 @@ func rerun(buildpath string, args []string) (err error) {
 		binPath = filepath.Join(pkg.BinDir, binName)
 	}
 
+	binPath = strings.Replace(binPath, buildpath, filepath.Base(binName), -1)
```

Comment:
Ya, the binpath contain the library path but due to go's new system, that
is not possible. i.e
bin path was in the form:

  "C:\Users\Flux\Lab\go\cmd\bin\github.com\influx6\watch\watch"

which was wrong

On Wed, Sep 16, 2015 at 9:42 PM, John Asmuth <notifications@github.com>
wrote:

> In rerun.go
> <https://github.com/skelterjohn/rerun/pull/27#discussion_r39682295>:
>
> > @@ -183,6 +186,8 @@ func rerun(buildpath string, args []string) (err error) {
> >  		binPath = filepath.Join(pkg.BinDir, binName)
> >  	}
> >
> > +	binPath = strings.Replace(binPath, buildpath, filepath.Base(binName), -1)
>
> did you mean strings.Replace(binPath, binName, filepath.Base(binName), -1)
> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/skelterjohn/rerun/pull/27/files#r39682295>.
>